src/components/Avatar/Avatar.vue

```diff
@@ -65,22 +65,31 @@
 		<div v-if="userDoesNotExist" class="unknown">
 			{{ initials }}
 		</div>
-		<div v-if="hasMenu" v-show="contactsMenuOpenState"
-			class="popovermenu"
-			:class="`menu-${menuPosition}`">
-			<PopoverMenu :is-open="contactsMenuOpenState" :menu="menu" />
-		</div>
+		<Popover
+			v-if="hasMenu"
+			placement="auto"
```

Comment:
Should this be influenceable from the outside component?

Reply:
Don't think so